.github/workflows/reusable_build_tests_rust_pyo3.yml

```diff
@@ -193,9 +197,9 @@ jobs:
         with:
           python-version: '3.13'
       - name: Install Dependencies
-        run: pip install maturin pytest numpy setuptools
+        run: python -m pip install maturin pytest numpy setuptools
```

Comment:
Does this line still need setuptools? The corresponding line above doesn't use it